DeviceHiveESP8266.md

```diff
@@ -200,37 +204,59 @@ ssid=ssid&pass=pass&url=http%3A%2F%2Fexample.com%2Fapi&id=deviceid&key=accesskey
 
 # Pin definition
 
- Pin name in commands |  Function        |ESP8266 pin number |NodeMCU board pin
-----------------------|------------------|-------------------|------------------
-GPIO                  |                  |                   |
-      "0"             |    GPIO0         |        15         |       D3
-      "1"             |  GPIO1, UART_TX  |        26         |       D10
-      "2"             |    GPIO2         |        14         |       D4
-      "3"             |  GPIO3, UART_RX  |        25         |       D9
-      "4"             |    GPIO4         |        16         |       D2
-      "5"             |    GPIO5         |        24         |       D1
-      "12"            | GPIO12, SPI_MISO |        10         |       D6
-      "13"            | GPIO13, SPI_MOSI |        12         |       D7
-      "14"            | GPIO14, SPI_CLK  |        9          |       D5
-      "15"            |    GPIO15        |        13         |       D8
-ADC                   |                  |                   |
-      "0"             |    ADC0          |        6          |       A0
-Common                |                  |                   |
-      "0"             |          all pins in current group
-  
-_Notes:  
-GPIO6-GPIO11 usually connected to on-module EEPROM, that is why no API for this pins._
+| Pin name in commands |  Function        |ESP8266 pin number |NodeMCU board pin |
```

Comment:
it was broken. take a look at code-review branch https://github.com/Pilatuz/esp8266-firmware/blob/code-review/DeviceHiveESP8266.md#pin-definition